samples/tfm_integration/psa_firmware/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+# Override the binary used by qemu
+set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_merged.hex")
```

Comment:
This should be covered by the board cmake file(s) already:

- mps2_an521_ns: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mps2_an521/board.cmake#L15
- mps3_an547_ns: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mps3_an547/board.cmake#L24

Reply:
Done. Pushing soon.